app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.update({
+            name: self
+        })
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    persons = [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        if person.get("wife") and person["wife"] is not None:
+            Person.people[person["name"]].wife = (
+                Person.people)[person["wife"]]
+        if person.get("husband") and person["husband"] is not None:
+            Person.people[person["name"]].husband = (
+                Person.people)[person["husband"]]
```

Comment:
Similarly, ensure that `person["husband"]` is a valid key in the `Person.people` dictionary to avoid potential `KeyError`. Adding error handling or validation could be beneficial.